### ISSUE TITLE
Package fails to build because of FMT errors

### ISSUE THREAD

**Issue by HurricanePootis** [bartoszek/AUR-luxcorerender#14]
```
[1008/1060] Building CXX object src/luxcore/CMakeFiles/luxcore.dir/luxcoreimpl.cpp.o
FAILED: src/luxcore/CMakeFiles/luxcore.dir/luxcoreimpl.cpp.o 
/usr/bin/c++ -DBOOST_ALL_NO_LIB -DBOOST_FILESYSTEM_NO_DEPRECATED -DBOOST_PYTHON_STATIC_LIB -DBOOST_STATIC_LIB -DCPP_API_EXPORTS -DSPDLOG_FMT_EXTERNAL=1 -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/json-3.7.3/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/cuew/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/clew/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/optix-7.1.0/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/robin-hood-hashing-3.9.1/src/include -I/usr/include/python3.11 -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cloudproviders -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/build/generated/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/bcd-1.1/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/opencolorio-2.0.0/include -isystem /usr/include/Imath -isystem /usr/include/OpenEXR -march=znver3 -O3 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -w -std=c++11 -Wall -Wno-long-long -pedantic -msse -msse2 -msse3 -mssse3 -fPIC -fopenmp -std=gnu++14 -MD -MT src/luxcore/CMakeFiles/luxcore.dir/luxcoreimpl.cpp.o -MF src/luxcore/CMakeFiles/luxcore.dir/luxcoreimpl.cpp.o.d -o src/luxcore/CMakeFiles/luxcore.dir/luxcoreimpl.cpp.o -c /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp
In file included from /usr/include/spdlog/fmt/fmt.h:31,
                 from /usr/include/spdlog/common.h:50,
                 from /usr/include/spdlog/spdlog.h:12,
                 from /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/include/luxcore/luxcorelogger.h:25,
                 from /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp:24:
/usr/include/fmt/core.h: In instantiation of ‘constexpr fmt::v10::detail::value<Context> fmt::v10::detail::make_arg(T&) [with bool PACKED = true; Context = fmt::v10::basic_format_context<fmt::v10::appender, char>; T = const luxcore::Camera::CameraType; typename std::enable_if<PACKED, int>::type <anonymous> = 0]’:
/usr/include/fmt/core.h:1810:51:   required from ‘constexpr fmt::v10::format_arg_store<Context, Args>::format_arg_store(T& ...) [with T = {double, const char [87], const luxcore::Camera::CameraType}; Context = fmt::v10::basic_format_context<fmt::v10::appender, char>; Args = {double, char [87], luxcore::Camera::CameraType}]’
/usr/include/fmt/core.h:1828:18:   required from ‘constexpr fmt::v10::format_arg_store<Context, typename std::remove_cv<typename std::remove_reference<T>::type>::type ...> fmt::v10::make_format_args(T& ...) [with Context = basic_format_context<appender, char>; T = {double, const char [87], const luxcore::Camera::CameraType}]’
/usr/include/spdlog/logger.h:374:75:   required from ‘void spdlog::logger::log_(spdlog::source_loc, spdlog::level::level_enum, spdlog::string_view_t, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; spdlog::string_view_t = fmt::v10::basic_string_view<char>]’
/usr/include/spdlog/logger.h:90:13:   required from ‘void spdlog::logger::log(spdlog::source_loc, spdlog::level::level_enum, fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/usr/include/spdlog/logger.h:96:12:   required from ‘void spdlog::logger::log(spdlog::level::level_enum, fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/usr/include/spdlog/logger.h:158:12:   required from ‘void spdlog::logger::info(fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp:521:2:   required from here
/usr/include/fmt/core.h:1581:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1581 |       formattable,
      |       ^~~~~~~~~~~
/usr/include/fmt/core.h:1581:7: note: ‘formattable’ evaluates to false
[1009/1060] Building CXX object src/luxcore/CMakeFiles/pyluxcore.dir/luxcoreimpl.cpp.o
FAILED: src/luxcore/CMakeFiles/pyluxcore.dir/luxcoreimpl.cpp.o 
/usr/bin/c++ -DBOOST_ALL_NO_LIB -DBOOST_ATOMIC_NO_LIB -DBOOST_CHRONO_NO_LIB -DBOOST_FILESYSTEM_NO_DEPRECATED -DBOOST_FILESYSTEM_NO_LIB -DBOOST_IOSTREAMS_NO_LIB -DBOOST_NUMPY_NO_LIB -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_PYTHON_NO_LIB -DBOOST_PYTHON_STATIC_LIB -DBOOST_REGEX_NO_LIB -DBOOST_SERIALIZATION_NO_LIB -DBOOST_STATIC_LIB -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_NO_LIB -DCPP_API_EXPORTS -DFMT_SHARED -DSPDLOG_FMT_EXTERNAL=1 -Dpyluxcore_EXPORTS -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/json-3.7.3/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/cuew/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/clew/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/optix-7.1.0/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/robin-hood-hashing-3.9.1/src/include -I/usr/include/python3.11 -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cloudproviders -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/build/generated/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/bcd-1.1/include -I/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/deps/opencolorio-2.0.0/include -isystem /usr/include/Imath -isystem /usr/include/OpenEXR -march=znver3 -O3 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -w -std=c++11 -Wall -Wno-long-long -pedantic -msse -msse2 -msse3 -mssse3 -fPIC -fopenmp -std=gnu++14 -fPIC -MD -MT src/luxcore/CMakeFiles/pyluxcore.dir/luxcoreimpl.cpp.o -MF src/luxcore/CMakeFiles/pyluxcore.dir/luxcoreimpl.cpp.o.d -o src/luxcore/CMakeFiles/pyluxcore.dir/luxcoreimpl.cpp.o -c /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp
In file included from /usr/include/spdlog/fmt/fmt.h:31,
                 from /usr/include/spdlog/common.h:50,
                 from /usr/include/spdlog/spdlog.h:12,
                 from /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/include/luxcore/luxcorelogger.h:25,
                 from /home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp:24:
/usr/include/fmt/core.h: In instantiation of ‘constexpr fmt::v10::detail::value<Context> fmt::v10::detail::make_arg(T&) [with bool PACKED = true; Context = fmt::v10::basic_format_context<fmt::v10::appender, char>; T = const luxcore::Camera::CameraType; typename std::enable_if<PACKED, int>::type <anonymous> = 0]’:
/usr/include/fmt/core.h:1810:51:   required from ‘constexpr fmt::v10::format_arg_store<Context, Args>::format_arg_store(T& ...) [with T = {double, const char [87], const luxcore::Camera::CameraType}; Context = fmt::v10::basic_format_context<fmt::v10::appender, char>; Args = {double, char [87], luxcore::Camera::CameraType}]’
/usr/include/fmt/core.h:1828:18:   required from ‘constexpr fmt::v10::format_arg_store<Context, typename std::remove_cv<typename std::remove_reference<T>::type>::type ...> fmt::v10::make_format_args(T& ...) [with Context = basic_format_context<appender, char>; T = {double, const char [87], const luxcore::Camera::CameraType}]’
/usr/include/spdlog/logger.h:374:75:   required from ‘void spdlog::logger::log_(spdlog::source_loc, spdlog::level::level_enum, spdlog::string_view_t, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; spdlog::string_view_t = fmt::v10::basic_string_view<char>]’
/usr/include/spdlog/logger.h:90:13:   required from ‘void spdlog::logger::log(spdlog::source_loc, spdlog::level::level_enum, fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/usr/include/spdlog/logger.h:96:12:   required from ‘void spdlog::logger::log(spdlog::level::level_enum, fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/usr/include/spdlog/logger.h:158:12:   required from ‘void spdlog::logger::info(fmt::v10::format_string<T ...>, Args&& ...) [with Args = {double, const char (&)[87], const luxcore::Camera::CameraType&}; fmt::v10::format_string<T ...> = fmt::v10::basic_format_string<char, double, const char (&)[87], const luxcore::Camera::CameraType&>]’
/home/hurricane/.cache/paru/clone/luxcorerender/src/LuxCore-luxcorerender_v2.6/src/luxcore/luxcoreimpl.cpp:521:2:   required from here
/usr/include/fmt/core.h:1581:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1581 |       formattable,
      |       ^~~~~~~~~~~
/usr/include/fmt/core.h:1581:7: note: ‘formattable’ evaluates to false
[1023/1060] Building CXX object src/luxcore/CMakeFiles/luxcore.dir/__/__/generated/luxparser/luxparse.cpp.o
ninja: build stopped: subcommand failed.
```

**Comment by DarkShadow44** [comment]
Still doesn't compile for me using the latest update

**Comment by bartoszek** [comment]
@DarkShadow44 @HurricanePootis fixed in dea08789bcc68e25f00d1fae23671985452aec2e
